src/function/table/arrow_conversion.cpp

```diff
@@ -5,9 +5,19 @@
 #include "duckdb/common/types/arrow_aux_data.hpp"
 #include "duckdb/function/scalar/nested_functions.hpp"
 
+namespace {
```

Comment:
You can just wrap the whole shebang on the `namespace duckdb {` no?

Reply:
The reason I used an unnamed namespace is that we don't need to pollute the duckdb namespace with this struct, it's only relevant here.
This just makes it static essentially.